documentation/book/con-custom-resources-status.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// assembly-custom-resources.adoc
+
+[id='con-custom-resources-status-{context}']
+= {ProductName} custom resource status
+
+The `status` property of a {ProductName}-specific custom resource is configured to capture the _current_ state of the resource and make that information available.
```

Comment:
```suggestion
The `status` property of a {ProductName}-specific custom resource publishes the _current_ state of the resource to users and tools needing that information.
```

Reply:
Slight tweak to tom's edit:

"needing" > "that need".